.gitignore

```diff
@@ -0,0 +1,8 @@
+
+.swiftpm/xcode/package.xcworkspace/xcuserdata/elliotmawby.xcuserdatad/UserInterfaceState.xcuserstate
```

Comment:
Directory `.swiftpm/xcode/package.xcworkspace/xcuserdata/` should be ignored instead of specifically `elliotmawby`.

Maybe just ignore any folder with the name `/xcuserdata/` in the path?

Reply:
Feel free to merge after this fix

Reply:
sounds good /xcuserdata is being ignored now

Reply:
👍 Drop the 2 lines with `elliotmawby` and I think we are good to merge!